### DISCUSSION TITLE
Can't install ASReview (failed to build scikit-learn)

### DISCUSSION THREAD

**Discussion by W6296** [asreview/asreview#842] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
I tried to install ASReview via CMD in Windows and I get the following ERROR message:

![image](https://user-images.githubusercontent.com/94845214/142867896-728bfb5a-e4ec-4594-9aa3-1cc9b0d1b66d.png)

Can someone help with troubleshooting? Thanks.

**Comment by MeszarosAgota** [comment, marked as answer]
I had a similar problem yesterday, it also stated, that "Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/" 
<img width="677" alt="asreview_error" src="https://user-images.githubusercontent.com/71129673/143228944-f6fad844-4ed7-4df3-ba19-e817bdea06b1.png">

I downloaded it from the link above, I clicked the "Desktop development with C++" and a few of the optional files, and now ASReview works :)
(@mrmeszaros helped with my problems)

<img width="929" alt="asreview_visualstudio" src="https://user-images.githubusercontent.com/71129673/143228679-f74636e7-6ce3-46d7-bc26-c459f2266ae6.png">

@W6296 if you have the same problem, this might solve it, good luck! :)

**Reply by jteijema** [reply]
Yes, this is also the solution that worked for me. I followed this guide: [stackoverflow.com/questions/64261546/python-cant-install-packages](https://stackoverflow.com/questions/64261546/python-cant-install-packages). 

Thanks!

**Comment by yassenenzo** [comment]
Hi! 
I had this exact problem too. For me, the issue was that I had the wrong version of python. Your python version needs to be at least 3.7, but not 3.10 (there appear to be some issues with 3.10 and ASReview). You can check your python version by typing "python --version" in the cmd. 
Hope this helps!

**Reply by W6296** [reply]
Fixed with this solution! Thanks. 
Looking forward to a version of ASReview that works with the newest version of Python (3.10)
